product-owner/03-major-benefits-are-built-into-the-innersource-process-article.md

```diff
@@ -0,0 +1,38 @@
+In the InnerSource Commons, we talk about the concept of open in the lessons we've learned in open source and how we might reuse what we know or have learned in an enterprise environment. Let me step through a few of those from a manager's perspective. I'll talk about how they've benefited me as a director and my product managers-- that's what we call product owners at PayPal. 
+
+First of that is open code. What do I mean by open code? Basically the fact that code is visible by all of the company, and there's a process for other developers to be able to submit pull requests on other code bases and get them accepted. To understand this more, please see the videos about trusted committers and contributing agreements for more details. 
+
+Open code for a manager also means no more waiting or escalating to get bugs fixed or features implemented on other teams' code bases. You can start to implement and plan more effectively. Often, your team's problem, or feature, may not be the highest priority for that other team. You no longer have to lean on escalations to upper management and politics to get access to that team. Instead, you have more power to determine priorities with your team and have your dependencies on others. Sometimes it may take longer because of the learning curves. But if it is a team that is a constant bottleneck, you can get those stories out of a backlog that's been sitting there for years. 
```

Comment:
```suggestion
Open code for a manager also means no more waiting or escalating to get bugs fixed or features implemented on other teams' code bases. You can start to implement and plan more effectively. Often, your team's problem (or feature) may not be the highest priority for that other team. You no longer have to lean on escalations to upper management and politics to get access to that team. Instead, you have more power to determine priorities with your team and have your dependencies on others. Sometimes it may take longer because of the learning curves. But if it is a team that is a constant bottleneck, you can get those stories out of a backlog that's been sitting there for years. 
```